sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_search_sessions_v1/query.sql

```diff
@@ -0,0 +1,220 @@
+WITH events_unnested AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    timestamp AS event_timestamp,
+    normalized_channel,
+    normalized_country_code,
+    name AS event_name,
+    CASE
+      WHEN name = 'engagement'
+        AND (
+          (
+            mozfun.map.get_key(extra, "engagement_type") IN (
+              "click",
+              "drop_go",
+              "enter",
+              "go_button",
+              "paste_go"
+            )
+          )
+        )
+        THEN 'engaged'
+      WHEN name = 'abandonment'
+        THEN 'abandoned'
+      WHEN name = 'engagement'
+        AND (
+          (
+            mozfun.map.get_key(extra, "engagement_type") NOT IN (
+              "click",
+              "drop_go",
+              "enter",
+              "go_button",
+              "paste_go"
+            )
+          )
+        )
+        THEN "annoyance"
+      ELSE NULL
+    END AS search_session_type,
+    SPLIT(mozfun.map.get_key(extra, "results"), ',')[OFFSET(0)] AS result_type,
+    SPLIT(mozfun.map.get_key(extra, "results"), ',') AS results,
+    mozfun.map.get_key(extra, "selected_result") AS selected_results,
+    mozfun.map.get_key(extra, "engagement_type") AS engagement_type,
+    mozfun.map.get_key(extra, "n_chars") AS number_of_chars_typed,
+    mozfun.map.get_key(extra, "n_results") AS num_total_results,
+    metrics,
+    metrics.uuid.legacy_telemetry_client_id AS legacy_telemetry_client_id,
+    e.key AS experiment_id,
+    e.value.branch AS branch,
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.events_v1`,
+    UNNEST(events)
+  CROSS JOIN
+    UNNEST(ping_info.experiments) AS e
```

Comment:
I don't think we should unpack experiments. This will create duplicates, so that there are multiple rows for each search session. We should just leave experiments as is, as a nested field. The Looker team has ways of auto-unpacking experiments correctly, if needed. @dzeber , what do you think?

Reply:
Agreed. I fixed the query.sql and schema.yaml. 
Also, populated the table `moz-fx-data-shared-prod.analysis.urlbar_search_sessions_v1_ak` for 2023-07-01

Reply:
thanks!

Reply:
yeah, we don't need to unpack them. The `experiments` field just needs to be there so that this table can be used in Jetstream.